src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -40,8 +40,8 @@ pub struct TargetInfo {
     pub rustflags: Vec<String>,
     /// Extra flags to pass to `rustdoc`, see `env_args`.
     pub rustdocflags: Vec<String>,
-    /// Remove this when it hits stable (1.44)
-    pub supports_bitcode_in_rlib: Option<bool>,
+    /// Remove this when it hits stable (1.45)
```

Comment:
If https://github.com/rust-lang/rust/pull/71716 is backported to beta, shouldn't this still be 1.44?

Reply:
It's unfortunately a bit confusing, but this flag is also being used as a proxy for "rustc can read its own `-Clinker-plugin-lto` object files", which is functionality on nightly that is not being backported. Only the option rename is being backported to beta.

Reply:
Er actually turns out that the backport isn't necessary becuase `-Cbitcode-in-rlib` isn't even on beta.